### ISSUE TITLE
508 Accessibility: Color contrast - disabled calendar cell

### ISSUE THREAD

**Issue by grossb1** [department-of-veterans-affairs/va.gov-team#21852]
When reviewing the Project Cheetah workflow for color contrast the disabled calendar cell was flagged for failing a color contrast test.

.../health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/select-date-1

Steps to reproduce:
Navigate to one of the above pages
Activate your colorblind testing tool (I used the Colorblinding chrome plugin)
Sample the hex color of the disabled calendar cell day (You can use something like Pixie to do this)
Sample the hex color of the disabled calendar cell day background
Go to a website like https://whocanuse.com/ and enter the hex values sampled above
Notice that the contrast test fails.

@outerpress

**Comment by 1Copenut** [comment]
Please see https://github.com/department-of-veterans-affairs/va.gov-team/issues/21848#issuecomment-805969640 for this issue.
